### DISCUSSION TITLE
Do you have matrix groups or discord channel?

### DISCUSSION THREAD

**Discussion by Decodetalkers** [slint-ui/slint#2484] [General] [answered]
Maybe there is a group will be better

**Comment by bjorn** [comment, marked as answer]
From the `README.md`:

> We chat in [our Mattermost instance](https://chat.slint-ui.com/) where you are welcome to listen in or ask your questions.

**Comment by realroot2185** [comment]
It would be nice to have a matrix room too.
It can be bridged to mattermost I believe.
